src/main/java/study/racingCar/CarRacingGame.java

```diff
@@ -6,45 +6,27 @@
 
 public class CarRacingGame {
 
-    private ArrayList<Car> theWinners;
 
     public CarRacingGame() {
-        this.theWinners = new ArrayList<>();
     }
 
     public void playGame(int tryCount, Car[] cars){
         for (int i = 0; i < tryCount; i++) {
             roundGameResult(cars);
             System.out.println();
         }
-        Car winner = getTheWinner(cars);
-        getTheWinners(cars, winner);
-        printWinners();
+        Winner theWinner = new Winner();
+        printWinners(theWinner.createWinners(cars, theWinner.findTheWinner(cars)));
     }
 
-    private void printWinners(){
-        System.out.print("승자:: ");
+    private void printWinners(ArrayList<Car> theWinners){
```

Comment:
https://github.com/next-step/java-racingcar/pull/5156#discussion_r1395351370

이전 피드백과 동일한 피드백으로써 선언시 구현체 대신 인터페이스를 선언해보는 부분을 고민해보시면 좋을 것 같아요 😄 

```suggestion
    private void printWinners(List<Car> theWinners){
```